app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -115,5 +116,19 @@ class CustomsDeclareExportsConnector @Inject()(
         logger.debug(s"CUSTOMS_DECLARE_EXPORTS cancel declaration response is --> ${response.toString}")
         response
       }
+}
 
+object CustomsDeclareExportsConnector {
+  def toXml(metaData: MetaData): String = {
+    import java.io.StringWriter
+
+    import javax.xml.bind.{JAXBContext, Marshaller}
+
+    val jaxbMarshaller = JAXBContext.newInstance(classOf[MetaData]).createMarshaller
```

Comment:
This is quite heavy operation - can we move this Val to object and not build marshaller every time ?

Reply:
Ok - from [Internet](https://timjansen.github.io/jarfiller/guide/jaxb/multithreading.xhtml) we must not share marshaller but Context is perfectly fine.